src/App.tsx

```diff
@@ -484,12 +508,14 @@ function Row({
     ${className};
 
     & svg {
+      flex: 1;
     }
   `;
 
   const textStyle = css`
     margin-left: 6px;
     text-align: left;
+    flex: 10;
```

Comment:
Is there a reason for this to be 10?

Reply:
Mostly just that if I didn't specify the flex values for this (and the other child of the flex container: the svg), the SVG would shrink when there was a lot of text.  10:1 seems to preserve the original spacing/sizing (as seen in the "leaving this month" warning).